src/components/ProgressBar/index.tsx

```diff
@@ -83,29 +98,36 @@ const ProgressIndicator = styled.div`
 `;
 
 const ProgressStop = stopCreator`
-  border-radius: 12px;
+  border-radius: ${distance.small};
   position: absolute;
-  width: 12px;
-  height: 12px;
-  top: -4px;
-  background: ${themed(props => (props.active ? props.theme.fill : props.theme.textDisabled))};
-  border: 2px solid ${themed(props => props.theme.background)};
-  transform: translate(-50%, 0);
+  width: ${distance.small};
+  height: ${distance.small};
+  background: ${({ active }) => (active ? midnight : grey4)};
```

Comment:
Are these colors also available on the theme? Should we take them from the theme (uiX, textX or supportX)?

Reply:
We have `midnight ` used in button's theme, but it doesn't make sense to take it from there. And grey4 is actually not used in the theme.

Reply:
Well, these themes are deprecated and we do not want to use them (best case -> they are all removed). So I am exclusively referring to uiX, textX and supportX. Maybe we should introduce something like that on theme for midnight and grey4, because I can imagine that these will break consistency when custom themes are applied.

Reply:
I've added these two colors to theme, but we'll have to review the naming conventions with designers, since I don't like that we use `supportX` color in progress bar, I think the name should be more neutral if they use the same color for support and progress bar.

Reply:
Yes, I saw it, but wasn't actually sure which one to use, and that's why I just used support1, but anyways this will be ui7 now, and I'm checking with Emanuel which name to use for this grey one.